lib/StaticNarrative/exceptions.py

```diff
@@ -58,6 +59,4 @@ def __init__(self, ws_server_err, ws_id, message=None, http_code=500):
             self.message = ws_server_err.message
 
     def __str__(self):
-        return "WorkspaceError: {}: {}: {}".format(
-            self.ws_id, self.http_code, self.message
-        )
+        return f"WorkspaceError: {self.ws_id}: {self.http_code}: {self.message}"
```

Comment:
use some of them newfangled f-strings

Reply:
I think there's a few of those. Some of this was (rather embarrassingly) copied from an older version of the Narrative repo.

Reply:
Or other places.